content/Product/OpenshiftMonitoring/faq.md

```diff
@@ -0,0 +1,140 @@
+# Debugging the in-cluster monitoring stack
```

Comment:
Let's be specific, what exactly we are debugging. There might be many in clusters stacks for monitoring.

Something like 
```suggestion
# Debugging the default Openshift 4.x in-cluster Monitoring: Platform Monitoring and User Workload Monitoring Products
```